app/models/clear_cms/ability.rb

```diff
@@ -4,19 +4,26 @@ class ClearCMS::Ability
   def initialize(user)
     user ||= ClearCMS::User.new
 
-    can :manage, :all
+    if user.system_permission == 'alumni'
 
-    can :manage, :all if user.system_permission == 'administrator'
+      can :update, ClearCMS::User, :id => user.id
```

Comment:
we may want to just `can :read` this, or at least put a comment like the following before this line:
"NOTE: This allows alumni to edit their profile (including name, bio, and avatar) which may continue to appear on the site"